### DISCUSSION TITLE
feature: something like wilder.nvim but for lazyvim's UI

### DISCUSSION THREAD

**Discussion by shelper** [LazyVim/LazyVim#1060] [Q&A] [answered]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

wilder.nvim provides a fuzzy search for both cmdline and arguments to cmdline like below 
![image](https://gist.githubusercontent.com/gelguy/018d7fb1d5292500b2f9bc6d209a7972/raw/3be41ed723b5eb31928787ee25e5545c08a96061/border.gif)
i wonder if lazyvim can provide similar with its own ui setup

I tried to make it work but failed..

### Describe alternatives you've considered

None

**Comment by folke** [comment]
You could just add wilder to your own config. Although, I'm pretty sure it will clash with noice, so you'll have to disable that.
Alternatively there's also a cmdline source for cmp.

**Reply by shelper** [reply]
yeah, i tried to make cmdline work like wilder, but failed, it seems not possible, or i dont know enough about nvim and lua, newbie here ...

**Comment by dhupee** [comment, marked as answer]
hey @shelper, hope its not very late, and thanks to @folke in advanced for the amazing distro, I think I can give you solution which unfortunately will disabled folke's noice but wilder has UI replacement that isnt half bad if i judge it

![image](https://github.com/LazyVim/LazyVim/assets/32934122/cd37c6d1-31dc-4c74-99a9-b1f6548594bd)

dont mind the other tmux pane its just me looking at reference, so......what i did is to disabled noice, just return the plugin, enabled = false, pretty straightforward

the wilder however you can scroll on the github it provide one of the UI which is blend pretty nicely imo

so this is my wilder.lua

```lua
return {
  "gelguy/wilder.nvim",
  config = function()
    local wilder = require("wilder")
    wilder.setup({ modes = { ":", "/", "?" } })

    wilder.set_option(
      "renderer",
      wilder.popupmenu_renderer(wilder.popupmenu_border_theme({
        highlights = {
          border = "Normal", -- highlight to use for the border
        },
        -- 'single', 'double', 'rounded' or 'solid'
        -- can also be a list of 8 characters, see :h wilder#popupmenu_border_theme() for more details
        border = "rounded",
      }))
    )
  end,
}
```

Because by no mean to offend folke but the noice is kinda get in the way and too big to my liking and since wilder has this option, and folke is so generous to have us easily disable the preinstalled plugin, well....the find string still in the bottom either so its quite nice to me

do i add something original? hell no, im not like you who contribute to tensorflow codebase(sorry i cant help it)

hope it helps mate
